### DISCUSSION TITLE
Error: @ vercel/edge-config: No connection string provided

### DISCUSSION THREAD

**Discussion by sdliriano** [sushi-labs/sushiswap#1423] [Q&A]
I cloned the SushiSwap repository, https://github.com/sushiswap/sushiswap, and ran the commands `pnpm install` which installed the dependencies and `pnpm exec turbo run dev --filter=evm` which runs the UI in development mode. The dev enviornment is super slow and whenever I try to visit the Swap page I get this error:

Error: @ vercel/edge-config: No connection string provided

I can visit most of the other pages in the site. Has anyone experienced this?

**Comment by SecretRecipe** [comment]
Same, anyone figured it out? I've made what I thought was the correct edge connection string on vercel, doesn't seem to accept it though..
